dask_kubernetes/core.py

```diff
@@ -154,6 +155,7 @@ class Scheduler(Pod):
 
     def __init__(self, idle_timeout: str, service_wait_timeout_s: int = None, **kwargs):
         super().__init__(**kwargs)
+        self.cluster._log("Creating scheduler pod on cluster. This may take some time.")
```

Comment:
I'm hitting a 
```
AttributeError: 'KubeCluster' object has no attribute '_log'
```
When instantiating a `KubeCluster`

Reply:
Thanks @Timost. This is dependent on an upstream change which will be released today. Hence why we haven't released here yet. If you are testing `dask-kubernetes` from master please also install `distributed` from master.

Reply:
Ha my bad ! Sorry for the noise.